plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,53 @@
+import discord
+from discord.ext import commands
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.diary_parser import diary_parser
+from plugins.TodayAtMun.Today import Today
+
+
+class TodayAtMun(AutomataPlugin):
+    def __init__(self, manifest, bot):
+        super().__init__(manifest, bot)
+        self.Tod = Today()
```

Comment:
```suggestion
        self.tod = Today()
```
an instance of today doesn't need to start with a capital letter, since itself is not a class

Reply:
Fixed!